packages/kujua-sms/tests/kujua-sms/utils.js

```diff
@@ -160,294 +162,6 @@ exports['getLabels'] = function(test) {
     test.done();
 };
 
-exports['getValues no clinic'] = function(test) {
```

Comment:
I couldn't find it being used any more so removed the method and the related tests.